e2e/scenario/scenario_ubuntu1804.go

```diff
@@ -9,6 +13,11 @@ func (t *Template) ubuntu1804() *Scenario {
 			ClusterSelector: NetworkPluginKubenetSelector,
 			ClusterMutator:  NetworkPluginKubenetMutator,
 			VHDSelector:     t.Ubuntu1804Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
```

Comment:
this scenario should just use what's already in the NBC, any new functionality should be tested separately in a new scenario(s)

Reply:
granted that template is pretty out of date :)